### ISSUE TITLE
Client output formatting

### ISSUE THREAD

**Issue by Updatebjarni** [Jolmberg/kakrafoon#1]
Each line ends with a lot of spaces. The width is fixed. Even with an 80-column terminal, if a field such as the user name field exceeds its normal width, the line extends across two lines on the terminal.

**Comment by Jolmberg** [comment]
Yes, we should check the width of the terminal and then format the output based on that. Take a look at the branch queue_formatting where I've taken a stab at that.
